pkg/products/solutionexplorer/reconciler.go

```diff
@@ -316,6 +322,35 @@ func (r *Reconciler) reconcileBlackboxTarget(ctx context.Context, installation *
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+func (r *Reconciler) addHSTSAnnotationToRoute(ctx context.Context, serverClient k8sclient.Client, inst *integreatlyv1alpha1.RHMI) (integreatlyv1alpha1.StatusPhase, error) {
+	upsRoute := &routev1.Route{
```

Comment:
One copy&paste mistake snicked in here :)
The variable should be renamed, since it's SolEx route, not ups.

Reply:
I pushed a commit that fixes this :)

Reply:
Thanks! :smiley: